content/200-concepts/100-components/01-prisma-schema/06-relations/07-referential-actions.mdx

```diff
@@ -8,6 +8,401 @@ preview: true
 
 <TopBlock>
 
-In [2.26.0](https://github.com/prisma/prisma/releases/tag/2.26.0) and later, you can define **referential actions** (Preview) on your relation fields. Official documentation is in progress - see [the 2.26.0 release notes for upgrade paths and how to use referential actions](https://github.com/prisma/prisma/releases/tag/2.26.0).
+In [2.26.0](https://github.com/prisma/prisma/releases/tag/2.26.0) and later, you can define **referential actions** on your relation fields. Referential actions determine what should happen to a record when a related record is deleted or updated. 
 
-</TopBlock>
\ No newline at end of file
+<Admonition type="warning">
+
+Referential actions are currently a **preview feature**.
```

Comment:
```suggestion
Referential actions are currently a **Preview feature**.
```

Teeny tiny consistency change.

Reply:
I don't understand this, why is it capitalized?